sources/api/host-containers/src/main.rs

```diff
@@ -434,22 +434,48 @@ where
     Ok(())
 }
 
+fn is_restart_required(settings: &Vec<&str>, container_name: &str) -> bool {
```

Comment:
```suggestion
fn is_restart_required(settings: &[&str], container_name: &str) -> bool {
```
Rust allows you to pass a `Vec` to a function taking a slice `&[]`, but this way you can also pass a real slice, so it's more flexible.